### PR TITLE
Add About section in Settings Dialog

### LINKED ISSUE #328: Display Akira version
I'm testing a few different instances of Akira on my system and it would be useful to have the version shown in the GUI somewhere, usually this can be found in an 'About' dialogue, or sometimes thrown into a corner of a system bar.

### PR DESCRIPTION
**Pull request by Alecaddd**
## Summary / How this PR fixes the problem?
Add an `About` section in the Settings Dialog.

## Steps to Test
Open the Settings Dialog and access the About Tab.

## Screenshots 
Screenshot from 2020-04-16 19-50-10

## This PR fixes/implements the following **bugs/features**:
- Add About dialog.
- Show Akira version from meson file.
- Implement an MD file to list all the Patreon supporters.

- Fixes #328

### CODE REVIEW

File: src/Dialogs/SettingsDialog.vala [hunk 1]
```diff
@@ -186,6 +183,96 @@ public class Akira.Dialogs.SettingsDialog : Gtk.Dialog {
...
     }
 
+    private Gtk.Widget get_about_box () {
+        var grid = new Gtk.Grid ();
+        grid.row_spacing = 6;
+        grid.column_spacing = 12;
+        grid.halign = Gtk.Align.CENTER;
+
+        var app_icon = new Gtk.Image ();
+        app_icon.gicon = new ThemedIcon ("com.github.akiraux.akira");
```
Comment (bilelmoussaoui) [inline src/Dialogs/SettingsDialog.vala:193]: We can use `Config.APP_ID` here

Comment (albfan) [reply src/Dialogs/SettingsDialog.vala:193]: Definitely! Icon is not render on flatpak

Captura de pantalla de 2020-04-17 10-11-18

**Comment by Alecaddd** [comment]
@albfan @bilelmoussaoui, patch updated, can you confirm the icon is visible now?

**Comment by bilelmoussaoui** [comment]
Works fine here, I totally forgot there was a "development" version of the icon :D 
image

**Pull request merged**